tutorials/colors/colorbar_only.py

```diff
@@ -84,9 +84,8 @@
 # --------------------------------------
 #
 # Here we illustrate the use of custom length colorbar extensions, used on a
-# colorbar with discrete intervals. To make the length of each extension the
-# same as the length of the interior colors pass the *extendfrac* argument as
-# *auto*.
+# colorbar with discrete intervals. To make the length of each extension same
+# as the length of the interior colors, use extendfrac='auto'.
```

Comment:
sorry to be picky, but I meant adding in double backticks around it, so:

```
``extendfrac='auto'``
```

I'd edit the branch if I could but I don't have commit rights :-/